backend/middleware/school.js

```diff
@@ -0,0 +1,51 @@
+let school = require("../db/school")
+
+const addNewClass = (req,res)=>{
+    
+    let name = req.body.name
+    let teacher = req.body.teacher
+    console.log(name, teacher, school)
+    if(!school.classes[name]){
+
+        let newClass = school.addClass(name, teacher)
+        res.json({
+            class: newClass,
+            message:"Class Created"
+
+        })
+        //console.log(mySchool.classes)
+    } else{
+        res.json({
+            error: "Class already Exists"
+        })
+    }
+}
+
+const getAllClass = (req, res, next) =>{
+    try{
+        let classes = school.classes
+    res.json({
+        classes
+    })
+    }catch(err){
+        next(err)
+    }
+}
+
+
+const addStudent = (req, res, next) =>{
+    try{
+        let newStudent = req.body
+        let classRegistration = req.params["className"]
+        school.enrollStudent(classRegistration, newStudent)
+        res.json({
+            status: newStudent,
```

Comment:
You are sending your request back as your response. This is not correct. You need to return `newStudent`/`addedStudent` in your `School.js` file, then assign the return value to a variable here in this file. That variable will hold the value of the new student that was just added in your backend. And that variable is what you should include in your `res.json`